implants/lib/eldritch/src/sys.rs

```diff
@@ -59,36 +61,86 @@ impl<'v> UnpackValue<'v> for SysLibrary {
 #[starlark_module]
 fn methods(builder: &mut MethodsBuilder) {
-    fn exec<'v>(this: SysLibrary, starlark_heap: &'v Heap, path: String, args: Vec<String>, disown: Option<bool>) -> anyhow::Result<Dict<'v>> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+    fn exec<'v>(
+        this: SysLibrary,
+        starlark_heap: &'v Heap,
+        path: String,
+        args: Vec<String>,
+        disown: Option<bool>,
+    ) -> anyhow::Result<Dict<'v>> {
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         exec_impl::exec(starlark_heap, path, args, disown)
     }
     fn get_os<'v>(this: SysLibrary, starlark_heap: &'v Heap) -> anyhow::Result<Dict<'v>> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         get_os_impl::get_os(starlark_heap)
     }
     fn dll_inject(this: SysLibrary, dll_path: String, pid: u32) -> anyhow::Result<NoneType> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         dll_inject_impl::dll_inject(dll_path, pid)
     }
     fn get_ip<'v>(this: SysLibrary, starlark_heap: &'v Heap) -> anyhow::Result<Vec<Dict<'v>>> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         get_ip_impl::get_ip(starlark_heap)
     }
     fn is_linux(this: SysLibrary) -> anyhow::Result<bool> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         is_linux_impl::is_linux()
     }
     fn is_windows(this: SysLibrary) -> anyhow::Result<bool> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
+        if false {
+            println!(
+                "Ignore unused this var. _this isn't allowed by starlark. {:?}",
+                this
+            );
+        }
         is_windows_impl::is_windows()
     }
     fn is_macos(this: SysLibrary) -> anyhow::Result<bool> {
-        if false { println!("Ignore unused this var. _this isn't allowed by starlark. {:?}", this); }
```

Comment:
Lets keep these as a single line don't want to take up so much space with dead code.
If there's a better way to suppress the compiler warnings that is probably a better solution.